packages/styled-components/transform/tests/fixtures/transpile-css-prop/output.js

```diff
@@ -120,15 +120,49 @@ const ObjectPropMixedInputs = (p)=>{
     </_StyledP13>;
 };
 const ObjectPropWithSpread = ()=>{
-    const css = {
+    const css1 = {
         color: 'red'
     };
     const playing = true;
-    return <_StyledDiv $_css13={css} $_css14={playing ? {
+    return <_StyledDiv $_css13={css1} $_css14={playing ? {
         opacity: 0,
         bottom: '-100px'
     } : {}}/>;
 };
+const id = (x)=>x;
+const LocalCallInterpolation = (p)=>{
+    const color = "red";
+    return <_StyledP14 $_css15={id(color)}>
+
+      H
+
+    </_StyledP14>;
+};
+const LocalCallInterpolation2 = ()=>{
+    return <_StyledP15 >
+
+      H
+
+    </_StyledP15>;
+};
+const LocalCallCssHelperProp = (p)=>{
+    const color = "red";
+    return <_StyledP16 $_css16={id(color)}>
+
+      H
+
+    </_StyledP16>;
+};
+// https://github.com/vercel/next.js/issues/38914
+function Home() {
+    const { x  } = {
+        x: 4
+    };
+    return <_StyledDiv2 $_css17={myCss(x)}/>;
+}
+const myCss = (x)=>css`
```

Comment:
What was the previous output?

Reply:
<img width="982" alt="image" src="https://user-images.githubusercontent.com/7829098/196859995-7eb0755a-b73b-4193-9a46-d0ece8a23bcb.png">
<img width="834" alt="image" src="https://user-images.githubusercontent.com/7829098/196860030-8ee2a912-3eff-4f94-b45a-71fa67336ed2.png">

Reply:
Thank you!